charts/Controls.tsx

```diff
@@ -1058,6 +1071,20 @@ export class ControlsFooterView extends React.Component<{
             <SettingsMenu chart={chart} onDismiss={this.onSettingsMenu} />
         )
 
+        const relatedQuestion = hasRelatedQuestions && (
+            <div className="relatedQuestion">
```

Comment:
nit: I think we're usually using kebab-case for CSS classes, i.e. `related-question`

Reply:
That would be my preference too but out of consistency with some of the neighbouring Controls classnames, I went with camelCase.

Reply:
Yeah the codebase is inconsistent. I try to use the kebab too, except when surrounded by camels.